isisd/isis_cli.c

```diff
@@ -1853,7 +1853,7 @@ void cli_show_isis_frr_lfa_load_sharing(struct vty *vty,
 					const struct lyd_node *dnode,
 					bool show_defaults)
 {
-	if (!yang_dnode_get_bool(dnode, NULL))
```

Comment:
doesn't it matter what the value of the boolean is?

Reply:
I'm not sure I get the question. Yes, the value matters and `yang_dnode_get_bool` returns that value :)
The change is because we need to output "no" when this value is `true` instead of `false`.

Reply:
of course, sorry - misread the api!